backend/webapi/Features/AccessRequests/ProviderReportingPortal.cs

```diff
@@ -62,15 +62,6 @@ public async Task<IDomainResult> HandleAsync(Command command)
                 return DomainResult.Failed();
             }
 
-            var prpAuthorization = await this.context.PrpAuthorizedLicences
-                .SingleOrDefaultAsync(authorizedLicence => filteredPlrDigest.LicenceNumbers.Contains(authorizedLicence.LicenceNumber));
```

Comment:
Now that we have removed this line, we no longer need to declare the filteredPlrDigest variable on line 54 since we are only using it once.

we can just do  
```C#
if (dto.AlreadyEnroled
    || !(await this.plrClient.GetStandingsDigestAsync(dto.Cpn))
        .With(AllowedIdentifierTypes)
        .HasGoodStanding)
{
    ...
```

like in other access requests.

Reply:
Fixed.